src/init/init.ts

```diff
@@ -230,7 +230,7 @@ function createSelectPrompt(env: YeomanEnvironment): InquirerQuestion {
  * error will be thrown.
  */
 export async function runGenerator(
-    generatorName: string, options: {[name: string]: any}): Promise<void> {
+    generatorName: string, options?: {[name: string]: any}): Promise<void> {
```

Comment:
Can ignore for now, but I prefer doing `options: {[name: string]: any} = {}` and removing the next line.

Reply:
+1, fixed since I needed to resolve the `.only` issue too